Model/src/main/java/org/gusdb/wdk/model/user/dataset/event/repo/UserDatasetOwnerRepo.java

```diff
@@ -0,0 +1,38 @@
+package org.gusdb.wdk.model.user.dataset.event.repo;
+
+import java.sql.Types;
+import javax.sql.DataSource;
+
+import org.gusdb.fgputil.db.runner.SQLRunner;
+
+/**
+ * Repo for DB operations on the "userdatasetowner" table.
+ */
+public class UserDatasetOwnerRepo
```

Comment:
again, about "Repo"... how about "DbUtils"  or "DbOperations"

Reply:
Dao? They are one class per table

Reply:
They have been updated with the suffix "DBActions"